app/assets/stylesheets/responsive/_wizard_layout.scss

```diff
@@ -63,3 +63,32 @@
 .wizard__next-step {
   display: block;
 }
+
+
+.js-wizard {
+  /*
+   * if JS is available, let's limit the height of the first question, which asks
+   * the question about which exemptions apply
+   */
+
+  [data-block="exemption"] fieldset {
+    max-height: 10em; 
+    transition: max-height 0.2s ease-out;
+    overflow: hidden;
+   }
+
+  [data-block="exemption"].expanded fieldset {
+    /*
+    * We're toggling the .expanded class with JS
+    */
+    max-height: 50em;
+    transition: max-height 0.2s ease-out;
+  }
+
+  .maximise-questions {
+    position: relative;
+    left: 50%;
+    transform: translateX(-50%);
+    border: 0;
+  }
+}
```

Comment:
Needs a newline at the end of the file. https://thoughtbot.com/blog/no-newline-at-end-of-file#following-the-rules-in-your-editor might be useful.

Reply:
Thanks, will add this